src/main/java/com/salesforce/einsteinbot/sdk/client/BasicChatbotClientImpl.java

```diff
@@ -86,11 +96,34 @@ void setHealthApi(HealthApi healthApi) {
     this.healthApi = healthApi;
   }
 
+  @VisibleForTesting
+  void setVersionsApi(VersionsApi versionsApi) {
+    this.versionsApi = versionsApi;
+  }
+
   @Override
   public BotResponse startChatSession(RequestConfig config,
       ExternalSessionId sessionId,
       BotSendMessageRequest botSendMessageRequest) {
 
+    String currentApiVersion;
```

Comment:
Can we move this logic to a separate method ?

Reply:
Yep, refactoring this